bot/exts/evergreen/issues.py

```diff
@@ -180,12 +180,6 @@ async def issue(
     @commands.Cog.listener()
     async def on_message(self, message: discord.Message) -> None:
         """Command to retrieve issue(s) from a GitHub repository using automatic linking if matching <repo>#<issue>."""
-        if not(
-            message.channel.category.id in WHITELISTED_CATEGORIES
-            or message.channel.id in WHITELISTED_CHANNELS_ON_MESSAGE
```

Comment:
Maybe we should remove the unused constant (`WHITELISTED_CHANNELS_ON_MESSAGE`)?

Reply:
They are still used in the `.issue` command.
https://github.com/python-discord/sir-lancebot/blob/dfe0e6001d4c1886e29b20eafacf4de1cc3a938b/bot/exts/evergreen/issues.py#L153-L156

Reply:
Not the one I mentioned.

Reply:
Oh right, my bad.

Reply:
They are in use, i just changed its name:
```diff
-WHITELISTED_CHANNELS_ON_MESSAGE = (
+WHITELISTED_CHANNELS_ISSUES = (
    Channels.organisation, Channels.mod_meta, Channels.mod_tools, Channels.staff_voice
)
```
https://github.com/python-discord/sir-lancebot/blob/3cf53c467837e0ebcc452142554264717680b301/bot/exts/evergreen/issues.py#L27-#L32

Reply:
Sorry but I don’t understand @Shivansh-007. Who changed what when, and what should this PR (#642) actually do?

Reply:
> Sorry but I don’t understand @Shivansh-007. Who changed what when, and what should this PR (#642) actually do?

1.  This is required by the `.issue` command (rework permission) PR, that is #638.
2. It is used to restrict the command to those white listed channels and few other white listed categories.
3. As the author of this feature, I would suggest to just remove the if...else check for channels/categories in the `on_message` listener.

Reply:
Alright, I'd rather not leave an unused constant lying around, I believe Git should properly handle this merge. If not, it will be up to the last merged PR to handle conflict.